src/utils/CredentialUtils.ts

```diff
@@ -0,0 +1,44 @@
+import {ICredential} from "@sphereon/ssi-types";
+import {ICredentialStatus} from "@sphereon/ssi-types/src/types/vc";
+
+import {CredentialStatusEnum, ICredentialSummary} from "../@types";
+
+class CredentialUtils {
```

Comment:
Please just create functions instead of creating classes with only static methods. This is not Java ;)

Reply:
Got it. Fixed.